OptimizelySDKCore/OptimizelySDKCore/Optimizely.m

```diff
@@ -628,6 +668,7 @@ - (void)handleErrorLogsForActivateUser:(NSString *)userId
                                      userInfo:errorDictionary];
     [self.errorHandler handleError:error];
     [self.logger logMessage:logMessage withLevel:OptimizelyLogLevelError];
+    return error;
 }
```

Comment:
I'd rather keep the callback decoupled because there is one case that I need to call the callback outside of handleErrorLogsForActivateUser:experiment